package.json

```diff
@@ -1,23 +1,26 @@
 {
-  "name": "kubernetes-secrets-plugin",
+  "name": "secrets-for-kubernetes",
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "build": "npm run build --workspaces",
+    "build": "turbo build",
+    "package": "npm run package --workspaces",
     "test": "npm run test --workspaces",
     "test:system": "npm run test:system --workspaces",
     "test:unit": "npm run test:unit --workspaces",
     "lint": "npm run lint --workspaces",
     "lintErrors": "npm run lintErrors --workspaces",
     "prepare": "husky install"
   },
+  "repository": "https://github.com/zowe/zowe-cli-secrets-for-kubernetes",
   "workspaces": [
     "./packages/*"
   ],
   "engines": {
-    "npm": ">=7.0.7",
-    "node": ">=14.17.3"
+    "npm": ">=8.19.3",
+    "node": ">=16.19.0"
   },
+  "packageManager": "npm@8.19.3",
```

Comment:
I changed them to these versions since node 14 will [end support in April](https://endoflife.date/nodejs), as for the `packageManager` property it seems to be a required property in order to use turbopack.